contracts/payment/Escrow.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.23;
+
+import "../math/SafeMath.sol";
+import "../ownership/Ownable.sol";
+
+
+/**
+ * @title Escrow
+ * @dev Base escrow contract, holds funds destinated to a payee until they
+ * withdraw them. The contract that uses the escrow as its payment method
+ * should be its owner, and provide public methods redirecting to the escrow's
+ * deposit and withdraw.
+ */
+contract Escrow is Ownable {
```

Comment:
Missing `initialize` call for `Ownable`

Reply:
No need, given that Escrow has no initialize, it inherits Ownable's